### ISSUE TITLE
Flag missing wikidata tag if object has wikipedia tag

### ISSUE THREAD

**Issue by doublah** [openstreetmap/iD#6990]
Currently, an object with a wikipedia tag but without a wikidata tag will render as any other object, iD should suggest to add the appropriate wikidata tag.

**Comment by dieterdreist** [comment]
if there is. Wikidata objects should represent the feature, while for wikipedia articles it is sufficient they are also about the feature. There isn't a 1:1 relation of wikidata and wikipedia, although in the beginning it generally was.

**Comment by matkoniecz** [comment]
is it really iD job to do so?

if yes, how interface would work to give any benefit over running an automated edit?

(for reference: I am running an automated edit adding missing `wikidata` and `wikipedia` if either is missing, if some region has mappers with consensus that it should be done also there I can run such edit in additional areas
https://wiki.openstreetmap.org/wiki/Mechanical_Edits/Mateusz_Konieczny_-_bot_account/adding_wikipedia_tags_based_on_wikidata_tags_in_Poland
https://wiki.openstreetmap.org/wiki/Mechanical_Edits/Mateusz_Konieczny_-_bot_account/adding_wikidata_tags_based_on_wikipedia_tags_in_Poland
)

**Comment by DujaOSM** [comment]
Frankly, now that Wikidata connects all language translations, directly linking Wikipedia articles from OSM is unnecessary and should be obsoleted (and it has a number of disadvantages, since it does not handle well all page moves on Wikipedia). I define `wikipedia` tags just because iD conveniently finds the appropriate `wikidata` entry for me (but it could do the same by querying Wikidata instead, frankly). 

So, it does make sense that iD offers to "upgrade" the objects with missing `wikidata` tag, just as it "upgrades" brand tagging.

**Comment by matkoniecz** [comment]
> So, it does make sense that iD offers to "upgrade" the objects with missing wikidata tag, just as it "upgrades" brand tagging.

So adding whatever `wikidata` value is linked to a given Wikipedia article? If that can and should be automatically with a bot edit, rather than in weird manual distributed bot edits wasting hours and days on people manually clicking validation rule in iD, often (typically?) without verification

> directly linking Wikipedia articles from OSM is unnecessary and should be obsoleted

personally I disagree, though it is not being proposed in this issue anyway

**Comment by DujaOSM** [comment]
I agree that iD validations should not become a replacement for QA tools such as Osmose or OSMInspector. The question is, where to draw the line? 

I suppose iD should issue warnings only for situations that you can plausibly (and inadvertently) create during iD edit sessions, not for every possible bad tag combination. This one does not really fall in that bucket: if I select a valid Wikipedia article from the UI, iD will automatically add a tag for its `wikidata` item. (There are situations where it won't work, e.g. some Wikipedia articles do not yet have a Wikidata entry, and you can cheat using raw tag editor). And there's always a possibility to fix such things by a bot run, as you suggested. 

So, I would agree with closing this as "not planned" -- it's fairly low-value for iD, better solved by other tools.

**Comment by matkoniecz** [comment]
> The question is, where to draw the line?

if workflow would end as "press upgrade tags button, no verification needed" and it exists thousands of times already, then it should be bot edit

**Comment by matkoniecz** [comment]
> I suppose iD should issue warnings only for situations that you can plausibly (and inadvertently) create during iD edit sessions

I entirely agree with it. And for cases where human evaluation is needed. Though there are traps here - with easy fixing of waterway/road intersection some mappers started adding fake fords/bridges/culverts as they were clicking random option to make warning go away

**Comment by tyrasd** [comment]
yeah, this seems like more like a job for a bot edit (if the community wants to do this in an automated way), or something like maproulette project, etc.
